ga4gh/frontend.py

```diff
@@ -195,13 +206,33 @@ def configure(configFile=None, baseConfig="ProductionConfig", extraConfig={}):
     theBackend.setDefaultPageSize(app.config["DEFAULT_PAGE_SIZE"])
     theBackend.setMaxResponseLength(app.config["MAX_RESPONSE_LENGTH"])
     app.backend = theBackend
+    app.secret_key = os.urandom(24)
+    app.oidcClient = oic.oic.Client()
+    if oidcConfig:
+        with open(oidcConfig) as oidcConfigFile:
+            configDict = yaml.load(oidcConfigFile)
+        app.oidcClient.redirect_uris = [configDict['redirecturi']]
+        app.oidcClient.provider_config(configDict['provider'])
+    if 'registration_endpoint' in app.oidcClient.provider_info:
+        app.oidcClient.client_info = app.oidcClient.register(
+            app.oidcClient.provider_info["registration_endpoint"],
+            redirect_uris=[configDict['redirecturi']])
+    else:
+        app.oidcClient.client_info = RegistrationResponse(
+            client_id=configDict['clientid'],
+            client_secret=configDict['clientsecret'],
+            verify_ssl=False)
+        app.oidcClient.client_id = configDict['clientid']
+        app.oidcClient.client_secret = configDict['clientsecret']
```

Comment:
I'm not sure that client_info is used anymore. I suspect it would work like this:

```python
if 'registration_endpoint' in app.oidcClient.provider_info:
        app.oidcClient.register(
            app.oidcClient.provider_info["registration_endpoint"],
            redirect_uris=[configDict['redirecturi']])
    else:
        response = RegistrationResponse(
            client_id=configDict['clientid'],
            client_secret=configDict['clientsecret'],
            verify_ssl=False)
        app.oidcClient.store_registration_info(response)
```